azure/components/get-aws-secrets-and-ssm-params.yml

```diff
@@ -10,7 +10,21 @@ steps:
         set -euo pipefail
         secret_name=$(expr match ${{ secret_id }} '.*/\(.*\)')
         secret_value=$(aws --profile apm_ptl secretsmanager get-secret-value --secret-id ${{ secret_id }} --query SecretString --output text)
-        echo "##vso[task.setvariable variable=${secret_name};issecret=true]${secret_value}"
+
+        export LINES=`echo "$secret_value" | wc -l`
+        echo "Number of lines secret contains: ${LINES}"
+
+        if (( $LINES > 1 )); then
```

Comment:
Hmm, we still need a handy way to get the full value as one thing... it would be good to read it in to a file somewhere and then have a parameter that holds the filename it's stored in

Reply:
Would this be in addition to the var per line or a replacement?

Reply:
In addition -- looks like you've done it